### ISSUE TITLE
Warnings and Detection for Mesh Transformation with Restart

### ISSUE THREAD

**Issue by psakievich** [Exawind/nalu-wind#1224]
If you run a restart but leave a mesh transformation in it will mess up the coordinates. We should have a check and print a warning, or make users specifically opt in to mesh transformation + restart.

**Comment by ashesh2512** [comment]
Yes, that is the intent of mesh transformation - change coordinates. Warning might be good, but would need people to weed through the code to catch it?

**Comment by psakievich** [comment]
I'm thinking we basically disable all mesh transformations on restart, unless the user specifically opts in with a flag in the transformation block?

**Comment by ashesh2512** [comment]
make sense

**Comment by psakievich** [comment]
I can't really think of a good reason to do a mesh transformation as part of a restart, but maybe someone will
